README.md

```diff
@@ -1,17 +1,42 @@
 # tic-tac-toe-ruby
 
-Third Ruby Microverse Project By Ángel Cussi &amp; Oscar Russi
+fifth Ruby Microverse Project By Ángel Cussi &amp; Oscar Russi
 
 ![screenshot](./screenshot.png)
 
-# Bubble sort on Ruby 
+# tic-tac-toe-ruby
 
-Third Microverse project for Ruby Curriculum
+fifth Microverse project for Ruby Curriculum
 
 # What it does
 
-- Created executable file to the project
-- Create folders bin and lib
+- tic tac toe the game for 2 player in the terminal mode
+
+# HOW TO PLAY
+
+- Tic tac toe is a game for two players, the game is played in a board of 3 x 3 cells
+- First players use X symbol
+- The second player uses 0 symbol
+- Each player can fill a cell with his/her symbol
+- the goal of both players is two add his/her symbol in the same row, column or one of the main diagonals
```

Comment:
Them meaning of the :point_up:  sentence is not clear to me. Can you refactor it please.